### PR TITLE
11.0 salesteam dashboard Fix kanban

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by kebeclibre** [comment]
@aab-odoo 
Indeed, the trigger record_updated works pretty well.
But, the field progressbar has an issue in this particular case,
which makes a second fix/test

**Comment by sswapnesh** [comment]
Fixes https://github.com/odoo/odoo/issues/25309

File: addons/web/static/src/js/fields/basic_fields.js [hunk 1]
```diff
@@ -2014,6 +2014,18 @@ var FieldProgressBar = AbstractField.extend({
...
         }
     },
+    /**
+     * The progress bar has more than one field/value to deal with
+     * i.e. max_value
+     *
+     * @override
+     * @private
+     */
+    _reset: function (record, event) {
```
Comment (aab-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: args not used

File: addons/web/static/src/js/fields/basic_fields.js [hunk 2]
```diff
@@ -2014,6 +2014,18 @@ var FieldProgressBar = AbstractField.extend({
...
+        this._super.apply(this, arguments);
+        this.max_value = this.recordData[this.nodeOptions.max_value] || this.max_value;
+    },
+
```
Comment (aab-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: unnecessary blank line

File: addons/web/static/src/js/fields/basic_fields.js [hunk 3]
```diff
@@ -2014,6 +2014,18 @@ var FieldProgressBar = AbstractField.extend({
...
+    /**
+     * The progress bar has more than one field/value to deal with
+     * i.e. max_value
+     *
+     * @override
+     * @private
+     */
+    _reset: function (record, event) {
+        this._super.apply(this, arguments);
+        this.max_value = this.recordData[this.nodeOptions.max_value] || this.max_value;
```
Comment (aab-odoo) [inline addons/web/static/src/js/fields/basic_fields.js]: this seems wrong... What if it is set to 0?

**Comment by aab-odoo** [comment]
@kebeclibre More than one commit... You have to write a PR description as merge commit message

**Pull request closed without merging**